crates/fluvio-extension-common/src/output/table.rs

```diff
@@ -57,20 +57,18 @@ where
 
         // Create the table
         let mut table = Table::new();
-        let mut format = *format::consts::FORMAT_CLEAN;
-        let pad_left = if indent { 5 } else { 1 };
-        format.padding(pad_left, 1);
-        table.set_format(format);
 
         // add header
-        table.set_titles(header);
+        table.set_header(header);
 
         // add rows
         for row in content {
             table.add_row(row);
         }
 
+        table.load_preset(comfy_table::presets::NOTHING);
+
         // print table to stdout
-        table.printstd();
+        table.print_std(2);
```

Comment:
I think that you should use `let pad_left = if indent { 5 } else { 1 };` instead of fixed `2`.

Reply:
My bad. Didn't see that.